stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/userreactions/UserReactionItem.kt

```diff
@@ -0,0 +1,96 @@
+package io.getstream.chat.android.compose.ui.components.userreactions
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.compose.previewdata.PreviewUserReactionData
+import io.getstream.chat.android.compose.state.userreactions.UserReactionItemState
+import io.getstream.chat.android.compose.ui.components.avatar.UserAvatar
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+
+/**
+ * Represent a reaction item with the user who left it.
+ *
+ * @param item The reaction item state.
+ * @param modifier Modifier for styling.
+ */
+@Composable
+public fun UserReactionItem(
```

Comment:
Maybe put this as a `DefaultUserReactionItem` and then put it internal and within the `UserReactions` file? 

That way we don't set up this expectation of having this fully customizable and all that

Reply:
I think we can still open this subcomponent